src/main/java/ch/njol/skript/effects/EffReplace.java

```diff
@@ -41,98 +41,118 @@
 import java.util.regex.Matcher;
 
 @Name("Replace")
-@Description("Replaces all occurrences of a given text with another text. Please note that you can only change variables and a few expressions, e.g. a <a href='../expressions.html#ExprMessage'>message</a> or a line of a sign.")
-@Examples({"replace \"<item>\" in {textvar} with \"%item%\"",
-		"replace every \"&\" with \"§\" in line 1",
-		"# The following acts as a simple chat censor, but it will e.g. censor mass, hassle, assassin, etc. as well:",
-		"on chat:",
-		"	replace all \"kys\", \"idiot\" and \"noob\" with \"****\" in the message",
-		" ",
-		"replace all stone and dirt in player's inventory and player's top inventory with diamond"})
-@Since("2.0, 2.2-dev24 (replace in multiple strings and replace items in inventory), 2.5 (replace first, case sensitivity)")
+@Description("Replaces all occurrences of a given text/regex with another text. Please note that you can only change " +
+	"variables and a few expressions, e.g. a <a href='/expressions.html#ExprMessage'>message</a> or a line of a sign.")
+@Examples({
+	"replace \"<item>\" in {_msg} with \"[%name of player's tool%]\"",
+	"replace every \"&\" with \"§\" in line 1 of targeted block",
+	"",
+	"# Very simple chat censor",
+	"on chat:",
+	"\treplace all \"kys\", \"idiot\" and \"noob\" with \"****\" in the message",
+	"\treplace using regex \"\\b(kys|idiot|noob)\\b\" with \"****\" in the message # Regex version for better results",
+	"",
+	"replace all stone and dirt in player's inventory and player's top inventory with diamond"
+})
+@Since("2.0, 2.2-dev24 (multiple strings, items in inventory), 2.5 (replace first, case sensitivity)," +
+	"INSERT VERSION (regex)")
```

Comment:
```suggestion
@Since("2.0, 2.2-dev24 (multiple strings, items in inventory), 2.5 (replace first, case sensitivity), INSERT VERSION (regex)")
```
I think this can be one line